### DISCUSSION TITLE
Make airflow notice a dag py file changed

### DISCUSSION THREAD

**Discussion by csm10495** [apache/airflow#20840] [Q&A] [answered]
Hey folks,

In Airflow 1.X I used to be able to do 

```
airflow db init
```

and then the webui would notice/pick-up on changes I made to a dag file. That doesn't seem to happen with Airflow 2. How can I make the running scheduler pickup on the airflow dag changing (and therefore rerender it in the webui)?

**Comment by marzlevane** [comment]
How about this one:

`python -c "from airflow.models import DagBag; d = DagBag();"`

**Reply by csm10495** [reply]
Didn't work :/ 

Sometimes the webui notices it for a few seconds, but then after enough browser refreshes it goes back to how it was before.

**Comment by jedcunningham** [comment, marked as answer]
You should see the changes relatively quickly. For smaller instances, somewhere between 30s-60s is normal.  If you aren't getting the changes even after waiting, seems like something may be wrong with your environment.

There is also a new command coming in Airfow 2.3.0 that will allow DAGs to be reserialized with `airflow dags reserialize` (see #19471).

**Reply by csm10495** [reply]
This is the 'true north' answer. The issue on our end seemed to be a weird configuration. 

We had:
```
SKIP_DAGS_PARSING=True
```

in some code just before the webserver would start. Once we removed that, things began updating as expected. Also it seems like a `airflow db init` would kick it 'now' instead of waiting for the config-defined interval.

Thanks all.
